### DISCUSSION TITLE
How to best get notified when Homebrew is fully ready for Macs with M1 running zsh

### DISCUSSION THREAD

**Discussion by rdfuhr** [Homebrew/discussions#404] [Getting started]
What is the best way to get notified when Homebrew is fully ready for Macs with the M1 chip that are running zsh?

I would prefer to wait until the installation script works transparently without having to do much, if any, manual tweaking.

I would also prefer to wait until Homebrew is fully compatible with the M1 chip.

**Comment by Rylan12** [comment]
I believe all the necessary pieces are there. If I remember, the install script has been updated to work on M1 macs (https://github.com/Homebrew/install/pull/373) and the `brew doctor` warning message was removed (https://github.com/Homebrew/brew/pull/10132).

We're in the process of updating our documentation: https://github.com/Homebrew/brew/pull/10182

I'd say the best way to keep track is to keep your eye on the [Homebrew blog](https://brew.sh/blog/). I'd imagine that there will be an announcement there when we officially support it. At a minimum, we'll post there when the next major or minor version of Homebrew is released. When we support, there almost certainly will be a message in that post.

**Reply by rdfuhr** [reply]
Thanks for your reply.  I imagine, as there are going to be an increasing number of M1 Macs out there, that many people will also be interested in knowing when M1 Macs are officially supported by Homebrew  

So, yes, mentioning the M1 compatibility update in your blog would be very helpful.

**Reply by Rylan12** [reply]
Just thought of one more place to look: we'll probably post a link to the blog post or separate announcement on our Twitter (https://twitter.com/MacHomebrew)
